plugins/formal-verification/testData/diagnostics/no_contracts/extension_function.kt

```diff
@@ -3,3 +3,42 @@ fun Int.<!VIPER_TEXT!>inc<!>(): Int = this + 1
 fun <!VIPER_TEXT!>extension_method_call<!>() {
     3.inc()
 }
+
+class Foo
+fun Foo.<!VIPER_TEXT!>nothing<!>(): Unit { }
+fun Foo.<!VIPER_TEXT!>nothing2<!>(a: Int): Unit { }
+fun Foo.<!VIPER_TEXT!>inside<!>(): Unit {}
```

Comment:
What distinguishes these functions?  This seems a bit redundant.
What does `nothing` mean?  They return unit.

Reply:
The first function is just an extension function that does nothing.

The second one is to test with parameters passing, since the first parameter of the function is of the receiver's type.

The third one is used for name collisions testing when defining an extension function with a class'scope.

Reply:
I renamed the functions from `nothing` to `return_unit`